docker/mxnet-model-server.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
```

Comment:
A higher level comment: this script exposes a control plane of sorts for MMS.
I can see how we can add another command for "load-model" that can instruct model server to load another key/value pair, or reload an existing model when the provided key already exists.
Not a comment about this PR, but something we can discuss offline.

Reply:
Sure. I was considering renaming the "restart" option in the MMS script to "reload", but I didn't change it because "restart" made sense in this context. For the hot-loading problem, I am still debating if it makes sense to have a "control plane API for a particular host instance (or add a new option to MMS script)" or have a "control plane API with each individual workers (or have another endpoint exposed like, ping, predict etc)" for doing loading/unloading of models. I will summarize this and we could collectively decide this offline and go forward in that direction.